src/pages/console/adminui/access-management.mdx

```diff
@@ -41,3 +37,44 @@ Follow these instructions to add and manage team members and their access to a p
 <Callout warning>
 If a team member logs into Amplify Studio, their login token is valid for 60 minutes, unless explicitly logged out. When you change a team member's permission from *full access* to *manage-only* or when you delete a team member's access, the team member can continue accessing Amplify Studio with their previously granted permissions until their token expires.
 </Callout>
+
+## Understanding how Studio manages user access
+
+<Callout warning>
+  The following resources are all managed by Studio so please **do not modify their settings, as they may affect your login experience**.
+</Callout>
+
+### User pool
+Studio manages user access using an Amazon Cognito User Pool in your account. You can invite up to 50,000 monthly users to Studio without cost.
+
+Studio manages user access using an Amazon Cognito User Pool in your account, named:
+- *amplify_backend_manager_APPID*.
+
+### IAM Roles
+In order to give the *Full access* and *Manage only* groups the necessary permissions, Studio creates 2 IAM roles, named:
+- *USERPOOLID_Full-access*
+- *USERPOOLID_Manage-only*
+
+### Cognito Identity Pool
+An Amazon Cognito Identity Pool is also created to vend AWS credentials that are tied to the *Full access* and *Manage only* groups, named:
+- *amplify_backend_manager_APPID*
+
+### Cognito Lambda triggers
+To provide a passwordless login experience from AWS Amplify Console to Amplify Studio, Studio creates 4 Cognito Lambda triggers *associated with the above-mentioned User Pool*, named:
+- *amplify-login-create-auth-challenge-SHORT_CODE*
+- *amplify-login-custom-message-SHORT_CODE*
+- *amplify-login-define-auth-challenge-SHORT_CODE*
+- *amplify-login-verify-auth-challenge-SHORT_CODE*
+
+### Recreating Studio managed resources
+If you happen to delete or otherwise mismanage any of the resources mentioned above, you can re-create them by first disabling and then re-enabling Studio for your Amplify Project on the Amplify management console.
+
+1. Sign in to the AWS Management Console and open AWS Amplify.
+2. Select your Amplify project with Amplify Studio enabled.
+3. In the navigation pane, choose **Amplify Studio settings**.
+4. Turn off **Enable Amplify Studio.**
+5. Turn on **Enable Amplify Studio.**
+
+<Callout warning>
+  Disabling and re-enabling Studio will delete the User pool managing user access into Studio, which will require re-inviting members you'd like to give access to your Amplify Project. Disabling and re-enabling Amplify Studio will not make any changes on your Amplify Project.
```

Comment:
Maybe we can say something along the lines like `Disabling and re-enabling Amplify Studio will remove and recreate the Amplify Studio managed User pool resource used to access your project, you will need to re-invite your users to provide access to your Amplify Project. Disabling and re-enabling Amplify Studio will not modify any resources on your Amplify Project`